python/pyproject.toml

```diff
@@ -275,3 +277,22 @@ typeCheckingMode = "strict"
 reportMissingTypeStubs = false
 reportMissingParameterType = false
 reportMissingTypeArgumet = false
+
+[tool.ruff]
+line-length = 140
+indent-width = 4
+include = ["whylogs/api/logger/experimental/logger/**/*.py"]
+select = ["E", "F", "I", "W"]
+
+[tool.ruff.isort]
+known-first-party = ["whylogs", "langkit"]
```

Comment:
Do we ever reference langkit in whylogs? I think it would be a circular dependency.

Reply:
I'll get rid, was copy pasta